### PR TITLE
Add view classes

### PR DESCRIPTION
**Pull request by Jolly5**
Let's see if this is going to work :o

### CODE REVIEW

**Comment by torkleyy** [comment]
It seems to me your `view` branch is at least partially based on `logic`. Could we merge #5 first and rebase this PR on top of it?

**Comment by torkleyy** [comment]
Here is how to rebase:

First, download the current upstream repo.
To do that, execute

```
git fetch --all
```

Next, you have to rebase your branch onto `upstream/master`. To do that, execute

```
git rebase -i upstream/master
```

while `-i` means interactive. This will take you into rebase mode where, once a conflict appears you will be notified about that and asked to fix it. Once you fixed it, `git add` these files and execute `git rebase --continue`.

For resolving conflicts see https://help.github.com/articles/addressing-merge-conflicts/

**Comment by torkleyy** [comment]
Nice to see you working on this! Can you say how much is left to implement?

File: src/main/java/model/HighscoreModel.java [hunk 1]
```diff
@@ -60,8 +60,12 @@ public void notifyObject(HighscoreObserver obj) {
     
     /**
      * You cannot add Entries that contain the CharSequence "::"
+<<<<<<< HEAD
```
Comment (torkleyy) [inline src/main/java/model/HighscoreModel.java]: It seems like you didn't rebase correctly.

**Comment by Jolly5** [comment]
Well... hard to say. By now nothing happens when you klick on a wrong answer. Also there is no such thing as a Highscores GUI. And of course the preferences make no sense as we have neither music nor sounds included.

**Comment by torkleyy** [comment]
> Well... hard to say. By now nothing happens when you klick on a wrong answer

Is that a limitation of the current view classes?

**Comment by Jolly5** [comment]
> Is that a limitation of the current view classes?

No. I just added it anyway. The FailureView now gets opened once you click on a wrong answer.
I'll be away eating now. o/

File: src/main/java/model/HighscoreModel.java (hunk 1 above)
Comment (Jolly5) [reply src/main/java/model/HighscoreModel.java]: Oops. I must have missed that one. Thanks for telling me

File: src/main/java/model/GameModel.java [hunk 2]
```diff
@@ -66,16 +63,42 @@ public Question getNextQuestion() {
      * @return true, if the given index is the index of the correct answer,
```
Comment (torkleyy) [inline src/main/java/model/GameModel.java]: Doc comment outdated.

File: src/main/java/view/PreferencesView.java [hunk 3]
```diff
@@ -3,16 +3,113 @@
...
+import javax.swing.*;
+
+public class PreferencesView extends JFrame implements PreferencesObserver {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 1L;
+
+    private final PreferencesView view = this;
```
Comment (torkleyy) [inline src/main/java/view/PreferencesView.java]: :sob: :joy: Yeah, well, looking forward to your fix!

**Review by torkleyy — COMMENTED** [review]
Looks pretty good so far!

**Pull request merged**